tasks.py

```diff
@@ -0,0 +1,857 @@
+        else:
+            run(
+                "apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common",
+                hide=not verbose,
+                warn=True,
+            )
+            res = run("cat /etc/issue", hide=not verbose, warn=True)
+            if "Debian" in str(res):
+                run(
+                    "curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -",
+                    hide=not verbose,
+                    warn=True,
+                )
+                run("apt-key fingerprint 0EBFCD88", hide=not verbose, warn=True)
+                run(
+                    'add-apt-repository \
+                    "deb [arch=amd64] https://download.docker.com/linux/debian \
+                    $(lsb_release -cs) \
+                    stable"',
+                    hide=not verbose,
+                    warn=True,
+                )
+                run(
+                    "apt install docker-ce docker-ce-cli containerd.io",
+                    hide=not verbose,
+                    warn=True,
+                )
+
+                if run("docker", hide=not verbose, warn=True).ok:
+                    run('echo "===>docker {}"'.format(MESSAGE_OK))
+                else:
+                    run('echo "===>docker {}"'.format(MESSAGE_FAILED))
+            elif "Ubuntu" in str(res):
+                run(
+                    "curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -",
+                    hide=not verbose,
+                    warn=True,
+                )
+                run("apt-key fingerprint 0EBFCD88", hide=not verbose, warn=True)
+                run(
+                    'add-apt-repository \
+                    "deb [arch=amd64] https://download.docker.com/linux/debian \
+                    $(lsb_release -cs) \
+                    stable"',
+                    hide=not verbose,
+                    warn=True,
+                )
+                run("apt install docker-ce", hide=not verbose, warn=True)
+
+                if run("docker", hide=not verbose, warn=True).ok:
+                    run('echo "===>docker {}"'.format(MESSAGE_OK))
+                else:
+                    run('echo "===>docker {}"'.format(MESSAGE_FAILED))
+            else:
+                run("echo {}".format(MESSAGE_WRONG_PLATFORM))
+
+    elif PLATFORM == "Darwin":
+        if run("command -v docker", hide=not verbose, warn=True).ok:
+            run('echo "===>docker {}"'.format(MESSAGE_ALREADY_INSTALLED))
+        else:
+            if run("brew cask install docker", hide=not verbose, warn=True).ok:
+                run('echo "===>docker {}"'.format(MESSAGE_OK))
+            else:
+                run('echo "===>docker {}"'.format(MESSAGE_FAILED))
+        run("open /Applications/Docker.app")
+
+    else:
+        run("echo {}".format(MESSAGE_WRONG_PLATFORM))
+
+    return None
```

Comment:
return None is implicit, don't need it

wherever you have return None at the end of a function you can get rid of it